glide.yaml

```diff
@@ -32,8 +32,8 @@ import:
   version: ^1
 - package: github.com/stretchr/testify
   version: master
-  # TODO: before cherami transport can leave the x/ package, cherami-client-go must >=1.0
 - package: github.com/uber/cherami-client-go
+  version: ^2
```

Comment:
Raising the top and bottom of the range of acceptable versions is a backward-incompatible change because any project that depends on `^2` indirectly via YARPC and `^1` of Cherami directly will fail to install with Glide. We could expand the range with `> 1.0.0, <= 3.0.0`. Cherami will still need to revert the breaking change in a version 1 patch release.